Contribute/dotnet/dotnet-contribute.md

```diff
@@ -173,7 +192,23 @@ You build any .NET Core snippet or sample using the .NET Core CLI, which can be
 
 3. Add a readme.md to the root directory of your sample.
 
-   This should include a brief description of the code, and refer people to the article that references the sample.
+   This should include a brief description of the code, and refer people to the article that references the sample. The top of the readme.md must have the metadata required for the [samples browser](https://docs.microsoft.com/samples). The header block should contain the following fields:
+
+   ```yml
+   ---
+   name: "really cool sample"
+   description: "Learn everything about this really cool sample."
+   page_type: sample
+   languages:
+     - csharp
+     - fsharp
+     - vbnet
+   products:
+     - dotnet-core
```

Comment:
Are there other products that samples might be for, that we should list identifiers for?